pages/common/pprof.md

```diff
@@ -0,0 +1,24 @@
+# pprof
+
+> Command-line tool for visualization and analysis analysis of profile data.
```

Comment:
Double use of `analysis`. Maybe this is what you meant?
```suggestion
> Command-line tool for visualization and analysis of profile data.
```

Reply:
thanks, I made a commit with this fix